src/Pythonize.cxx

```diff
@@ -459,7 +477,96 @@ PyObject* VectorIAdd(PyObject* self, PyObject* args, PyObject* /* kwds */)
         PyErr_SetString(PyExc_TypeError, "argument is not iterable");
     return nullptr;               // error already set
 }
+PyObject *recursive_vector_init(PyObject *self, Py_buffer *view, PyObject *result, int ndim)
+{
+    // PyObject *tmp_result = PyList_New(0);
+    if (ndim == 1)
+    {
+        if (!result)
+            return nullptr;
+
+        Py_ssize_t fillsz = view->len / view->itemsize;
+        PyObject *pb_call = PyObject_GetAttrString(self, "push_back");
 
+        for (Py_ssize_t i = 0; i < fillsz; i++)
+        {
+            int val = Get_IndexValue<int>(view, i);
+            PyObject *item = PyLong_FromLong(val);
```

Comment:
Mapping of `view->format` to typename (e.g. `'i'` -> '`int`'; see low-level view type_traits).

TConverter* conv = CreateConverter(type_returned);
PyObject* item = conv->FromMemory(view->buf + i*view->itemsize);
if (conv->HasState()) delete conv;